### DISCUSSION TITLE
Pim core test case

### DISCUSSION THREAD

**Discussion by Revathi23Vijay** [pimcore/pimcore#16660] [Q&A / Support] [answered]
Hello team,

I have started to writing test cases for pimcore 10.5.15 version. I am getting getContainer on null when my test case trying access configuration class methods.

I installed the phpunit, codeception packages .

I am very new to pimcore. Can someone please help me to address this issue.

**Comment by Revathi23Vijay** [comment]
Please help me I am getting this error when I access dB from test case.

Please guide me to write functional test case in pimcore

**Comment by MichaelBrauner** [comment, marked as answer]
You can modify the `SYMFONY_PHPUNIT_VERSION` to your needs: https://symfony.com/doc/current/components/phpunit_bridge.html#modified-phpunit-script

For me this helped: 

1. in phpunit.xml.dist:  
```xml
<!- delete this line -->
<env name="SYMFONY_PHPUNIT_VERSION" value="7.4" />
<!- add this line -->
<server name="SYMFONY_PHPUNIT_VERSION" value="9.6"/>
```

**Comment by kingjia90** [comment]
Resolved by https://github.com/pimcore/pimcore/pull/17840 Thank you both for reporting and the fix
